kedro-datasets/kedro_datasets/ibis/table_dataset.py

```diff
@@ -159,29 +133,16 @@ def connection(self) -> BaseBackend:
         return self._connection
 
     def load(self) -> ir.Table:
-        if self._filepath is not None:
-            if self._file_format is None:
-                raise NotImplementedError
-
-            reader = getattr(self.connection, f"read_{self._file_format}")
-            return reader(self._filepath, self._table_name, **self._load_args)
-        else:
-            return self.connection.table(self._table_name)
+        return self.connection.table(self._table_name)
 
     def save(self, data: ir.Table) -> None:
-        if self._table_name is None:
-            raise DatasetError("Must provide `table_name` for materialization.")
-
         writer = getattr(self.connection, f"create_{self._materialized}")
         writer(self._table_name, data, **self._save_args)
 
     def _describe(self) -> dict[str, Any]:
         return {
-            "filepath": self._filepath,
-            "file_format": self._file_format,
             "table_name": self._table_name,
             "backend": self._connection_config["backend"],
-            "load_args": self._load_args,
```

Comment:
This shouldn't be removed.

Reply:
We are not using load_args anywhere, should you still need this passed to the init ?